.env.example

```
@@ -0,0 +1,4 @@
+# Same as langwatch/langwatch/.env.example, plus:
+ADMIN_EMAILS=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)